src/layouts/components/Editor/components/ImageBubbleMenu/ResizeImagePopover.tsx

```diff
@@ -65,7 +65,10 @@ export const ResizeImagePopover = ({
     editor
       .chain()
       .focus()
-      .setImage({
+      .updateAttributes("image", {
```

Comment:
question: I had the impression that this wouldn't work if there are more than one image in the editor?

Reply:
could we hide this `updateAttributes` method and expose a `setWidth` method on the `Image` extension instead? 

more inline with the principle of least privilege + it's obvious to devs when reading the code for `Image` how the attributes are set

Reply:
> could we hide this `updateAttributes` method and expose a `setWidth` method on the `Image` extension instead?
> 
> more inline with the principle of least privilege + it's obvious to devs when reading the code for `Image` how the attributes are set

Thanks for the suggestion! Did the refactoring in 688d5a3e7bf09f0d97e8a4b8402aef05c68156d3

Instead of exposing `setWidth` method, I added a method to `setImageStyle` on the `image` extension which can be used to add a locally scoped `style` tag to the image node. It accepts a string style and added to the node. 

Thought this would be a more suitable method since 1) there could be future cases to add other attribute in `style` tag than just `width`, 2) `width` is a string attribute inside `style` tag, feels it's more appropriate to expose method to manipulate on tag level rather than a specific string value. 

@seaerchin wdyt?

Reply:
sure, sounds good! do take note that currently we give the type as `style: string`, which is very risky - we might wanna consider giving an explicit type like:
```ts
style: { width: number }
```
and the method itself does the conversion to string.

the benefits is that it stops us from making careless mistakes (is width specified as px? %? if we wanted to add colours, we can typecheck to a fixed universe of colours maybe blue/green only) and gives feedback to the devs too.

Reply:
sgtm! was only giving `backgroundColor` as an example, so we might wanna remove it @__@

Reply:
Yeah no worries, just removed it in 103897178315870ae1628dd281b0ef10b6d7f671